docs/release-cycle.md

```diff
@@ -0,0 +1,42 @@
+# Open Europa Release Cycle
+This document explains the release process followed by Open Europa project for its components.
+Open Europa releases its component following semantic versioning (https://semver.org/)
+
+There will be three types of releases planned: 
+
+- **MAJOR**: Incompatible API changes, very rare and planned in. 
+- **MINOR**: Adds backwards-compatible manner functionalities and bug fixes.
+- **PATCH**: Adds backwards-compatible bug / security fixes and can be deployed instantaneously. No new functionality will be introduced.
+
+Open Europa components release plan will not follow a fixed overall timeline and can happen in different planned slots. Each component roadmap and release plan will be made available to public.
+
+## Release preparation and testing
+
+OpenEuropa Drupal components releases will follow Drupal 8 core releases and will always be tested against:
+- Current stable Drupal 8 core minor release (n)
+- Previous supported Drupal 8 core minor release (n-1)
+- Development branch for next Drupal 8 core minor release 
+
+This will allow to follow the same support cycle as Drupal core and be better prepared for next minor releases as they occur.
+
+When a new Drupal core minor version is released, Open Europa team commits to have a release of all its supported components compatible with it in a fixed time frame (two sprints).
+
+It is predicted that the majority of changes will come in the form of MINOR and PATCH releases. However if there is a need to change API provided by the component in a way that it is not backwards compatible a new MAJOR release can be produced. 
```

Comment:
Missing comma - common introductory words like _however_ should be followed by a comma.